.github/ci_setup.sh

```diff
@@ -0,0 +1,30 @@
+# Detray library, part of the ACTS project (R&D line)
+#
+# (c) 2022 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+#
+# This script is meant to configure the build/runtime environment of the
+# Docker contaners that are used in the project's CI configuration.
+#
+# Usage: source .github/ci_setup.sh <platform name>
+#
+
+# The platform name.
+PLATFORM_NAME=$1
+
+# Set up the correct environment for the SYCL tests.
+if [ "${PLATFORM_NAME}" = "SYCL" ]; then
+   source /opt/intel/oneapi/setvars.sh
+   export CXX=`which clang++`
+   export SYCLCXX=`which dpcpp`
+   export SYCL_DEVICE_FILTER=host
+   # This is a hack to make Acts's "--coverage" flag work correctly with
+   # oneAPI 2021.2.0. It may be removed once we update to a version of oneAPI
+   # in the CI that does not have this issue.
+   export LDFLAGS="-L${CMPLR_ROOT}/linux/compiler/lib/intel64_lin -lirc"
```

Comment:
:confused: The text is a bit inconsistent with the code here.

Do we need this with oneAPI 2022.0 as well? Since you use `ghcr.io/acts-project/ubuntu2004_oneapi:v20` in this PR (as you should), which comes with the currently latest oneAPI version.

https://github.com/acts-project/machines/releases/tag/v20

Reply:
It was just brain-less copy and paste.. Removed it